### DISCUSSION TITLE
Unable to discover host machines

### DISCUSSION THREAD

**Discussion by Kamesh-369** [vishnu-ks/Cloud-migration#183] [Q&A] [answered]
Aws to Aws:
After the discovery phase, the host machine is not getting discovered while migrating from Aws to Aws.
After converting the root volume from GPT to MBR the instance goes to unhealthy stage.

Azure to Aws:
After the discovery phase, the host machine is not getting discovered while migrating from Azure to Aws.

**Comment by iamvishnuks** [comment, marked as answer]
Hi @Kamesh-369 ,
For answering your first question

> Aws to Aws:
After the discovery phase, the host machine is not getting discovered while migrating from Aws to Aws.
After converting the root volume from GPT to MBR the instance goes to unhealthy stage.

If the server which you have configured xmigrate doesn't have ssh access to those servers in aws which you want to migrate, then it won't be able to discover those servers. Could you please share the logs if this is not the case?
It is recommended to take the clone of the instance to migrate and then do the GPT to MBR conversion on the cloned instance. If you restart that instance then it goes to unhealthy stage because GPT is what is expected by AWS.


> Azure to Aws:
After the discovery phase, the host machine is not getting discovered while migrating from Azure to Aws.

It should be related to the network access as I mentioned above. Otherwise please share the logs.
